File: codeforces/cf545D_queue.py
# unihernandez22
# https://codeforces.com/problemset/problem/545/D
# greedy

input()
a = list(sorted(map(int, input().split())))
b = set(a)
t = []

for i in b:
    if a.count(i) > 1:
        t += [i, i]
    else:
        t += [i]

i = 0
suma = 0
while i < len(t) and len(t) > 0:
    if suma > t[i]:
        t.pop(i)
    else:
        suma += t[i]
        i += 1

print(len(t))

File: codeforces/cf545D_queue.rs
// https://codeforces.com/problemset/problem/545/D
// greedy
use std::io;

fn main() {
    let mut n = String::new();

    io::stdin()
        .read_line(&mut n)
        .unwrap();

    let n: i64 = n.trim().parse().unwrap();

    let mut line = String::new();

    io::stdin()
        .read_line(&mut line)
        .unwrap();

    let mut t: Vec<i64> =
        line
        .split_whitespace()
        .map(|x| x.parse().unwrap())
        .collect();

    t.sort_by(|a,b| b.partial_cmp(a).unwrap());

    let mut ans: usize = 0;
    let mut time = 0;

    while !t.is_empty() {
        let client = t.pop().unwrap();
        if time <= client {
            ans += 1;
            time += client;
        }
    }

    println!("{}", ans);
}

